perf/h.js

```diff
@@ -0,0 +1,26 @@
+import bench from 'nanobench'
+import { h } from '../src/h'
+
+bench("vnode with no child 1M times", b => {
+  b.start()
+  for (let i = 0; i < 10000000; i++) {
```

Comment:
Basically generate the function to it's not optimized and type feedback and all does leak to other tests.